docs/en/mongodb-sink.md

```diff
@@ -0,0 +1,12 @@
+### MongoDB
+
+To use the MongoDB sink add the following flag:
+```
+    --sink=mongo:?<MONGO_URI>
+```
+
```

Comment:
Could add the version supported for mongodb

Reply:
added